### PR TITLE
0.7.0

### PR DESCRIPTION
**Pull request by StratosCut**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zero_one/client",
-  "version": "0.5.1",
+  "version": "0.7.0-alpha.0",
```
Comment (ryu3sigma) [inline package.json:3]: wtf

File: src/accounts/margin/MarginWeb3.ts [hunk 2]
```diff
@@ -580,7 +751,7 @@ export default class Margin extends BaseAccount<Schema> {
       ooKey = oo.key;
       createOo = false;
     }
-
+    if (maxBaseQtyBn.toNumber() == 0) return "";
```
Comment (ryu3sigma) [inline src/accounts/margin/MarginWeb3.ts:754]: 1) Why just return ""?
2) use ===
3) should throw error

File: src/config.ts [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 export const ZO_FUTURE_TAKER_FEE = 0.001; // 10bps
 export const ZO_OPTION_TAKER_FEE = 0.001; // temporary value, not actual
 export const ZO_SQUARE_TAKER_FEE = 0.0015; // 15bps, not actual
 export const USDC_DECIMALS = 6;
 
+export const BASE_IMF_DIVIDER = 1000;
+export const MMF_MULTIPLIER = 2;
+export const USD_DECIMALS = 6;
 // == 01 CONSTANTS ==
 export const ZO_STATE_KEY = new PublicKey(
```
Comment (ryu3sigma) [inline src/config.ts:17]: Add a deprecation flag

File: package.json (hunk 1 above)
Comment (StratosCut) [reply package.json]: get over it

**Pull request merged**